missating/More-Recipes
# #154313481 feedback server side test

### What does this PR do?

This PR implements all feedback on the test for the server side api 
It breaks down test into multiple files for each server side api

### Description of Task to be completed?

Implement peer review feedback that will come from this PR

### How should this be manually tested? 

Click on the `Travis badge` that is embedded in this branch and see the result of the test

### Any background context you want to provide?

N/A

### What are the relevant pivotal tracker stories?

154313481 - Implement feedback on the server side test

### Screenshots (if appropriate)

Shows the total number of test 

<img width="685" alt="screen shot 2018-01-19 at 3 09 10 pm" src="https://user-images.githubusercontent.com/29452947/35154510-c3c50402-fd2a-11e7-87a1-a7c12edb5640.png">

### Questions:

N/A
